### PR TITLE
Add Table of Contents section into Readme.

### PR DESCRIPTION
**Pull request by stocarul**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,34 @@
 # snowbox [![Build Status](https://travis-ci.com/algotech/snowbox.svg?branch=master)](https://travis-ci.com/algotech/snowbox)[![codecov](https://codecov.io/gh/algotech/snowbox/branch/master/graph/badge.svg)](https://codecov.io/gh/algotech/snowbox)
 
-## Install
+Opinionated Redux abstractions for faster development.
```
Comment (stocarul) [inline README.md:3]: @lgnstn I suggest to add the following sentence as well: `Although it is created explicitly for React and React native, it works with any JavaScript framework that uses Redux.`. What do you think?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
 
-Install from the NPM repository using yarn or npm:
+## Motivation
```
Comment (stocarul) [inline README.md:5]: @lgnstn Shouldn't this pe `Problem` instead of `Motivation`?

File: README.md (hunk 1 above)
Comment (lgnstn) [reply README.md:3]: It is just partially true because some of its features use Hooks, which are React specific.

File: README.md (hunk 2 above)
Comment (lgnstn) [reply README.md:5]: I don't have a strong feeling for either of them. @ssergiuss?

**Comment by stocarul** [comment]
Recreated here: https://github.com/algotech/snowbox/pull/27

**Pull request closed without merging**